GPU/GPUTracking/Global/GPUChainTrackingTRD.cxx

```diff
@@ -38,11 +38,10 @@ int GPUChainTracking::RunTRDTracking()
   mRec->PushNonPersistentMemory(qStr2Tag("TRDTRACK"));
   SetupGPUProcessor(&Tracker, true);
 
-  for (unsigned int iTracklet = 0; iTracklet < mIOPtrs.nTRDTracklets; ++iTracklet) {
-    if (Tracker.LoadTracklet(mIOPtrs.trdTracklets[iTracklet], mIOPtrs.trdTrackletsMC ? mIOPtrs.trdTrackletsMC[iTracklet].mLabel : nullptr)) {
-      return 1;
-    }
+  if (mIOPtrs.trdTracklets == 0) {
```

Comment:
I don't understand this check. I would check if the number of tracklets = 0, we can return.
If the tracklet pointer is zero and the number is not, then it is a bug.